### PR TITLE
Tash is back for round 2

### PR DESCRIPTION
**Pull request by alinagator**

### CODE REVIEW

File: posts/signal-is-better-than-telegram.md [hunk 1]
```diff
@@ -11,28 +11,28 @@ layout: layouts/post.njk
 ## 1. Signal is a Not-For-Profit 😇
 Telegram is for profit 💰. I want the reassurance that the profit motive isn't there for my communications platform - [because we've seen where that leads to before](../goodbye-whatsapp/#😈-the-faustian-pact).
 
-## 2. Telegram Collects More Of your Data 🗄
-[Telegram collects more data on you](https://mybroadband.co.za/news/security/382036-whatsapp-vs-signal-vs-telegram-vs-facebook-the-data-apps-collect-about-you.html). Whereas Signal, when under the full force of a [FBI subpoena Signal declared what was happening publicly, and they could only share account creation date](https://arstechnica.com/tech-policy/2016/10/fbi-demands-signal-user-data-but-theres-not-much-to-hand-over/), because that is all they had.
+## 2. Telegram Collects More of your Data 🗄
```
Comment (alinagator) [inline posts/signal-is-better-than-telegram.md:14]: Style thing, but you're a bit inconsistent with what you capitalise in your (sub)headings. E.g. sometimes you capitalise 'is'/'the'/nouns/etc., sometimes you don't. I haven't made edits throughout, but just something to keep an eye on you want to look more consistent.

File: posts/signal-is-better-than-telegram.md [hunk 2]
```diff
@@ -11,28 +11,28 @@ layout: layouts/post.njk
...
 
 ## 4. Signals E2E is the best 🥇
-Signal's end-to-end encryption protocol is [regarded as best in class](https://www.wired.com/story/signal-encryption-protocol-hacker-lexicon/) and is widely regarded as far superior to Telegram's. The [security industry has heavily criticised Telegram's encryption model](https://security.stackexchange.com/a/49802/115711).
+Signal's end-to-end encryption protocol is [regarded as best in class](https://www.wired.com/story/signal-encryption-protocol-hacker-lexicon/), and is widely regarded as far superior to Telegram's. The [security industry has heavily criticised Telegram's encryption model](https://security.stackexchange.com/a/49802/115711).
 
 ## 5. Signal Protects You By Default 🔒
-All of Signal's messages and content are encrypted by default, not the case for Telegram.
+All of Signal's messages and content are encrypted by default. Not the case for Telegram.
 
 ## 6. Signal's Passes Independently Verified ✅
```
Comment (alinagator) [inline posts/signal-is-better-than-telegram.md:27]: What are "Signal's Passes"? Maybe should read "Signal is independently verified"?

File: posts/signal-is-better-than-telegram.md [hunk 3]
```diff
@@ -11,28 +11,28 @@ layout: layouts/post.njk
...
+Ok this sounds like a reason against Signal - hear me out! Signal's [group chats are limited to 1000](https://support.signal.org/hc/en-us/articles/360007319331-Group-chats), Telegram's are not. I can't think of a group I'd need to be directly messaging that had anywhere close to a thousand participants. Signal is not trying to be social media. I want my primary messaging app to be just that - do one thing, and do it well. The ability to communicate with more than 1000 people can help the growth of adverse political groups, for [example things like Proud Boys on Telegram 🤦‍♂️](https://t.me/s/proudboysusa) - I'd rather not see another assault on American democracy or feel ethically challenged because my messaging app enables such things.
```
Comment (alinagator) [inline posts/signal-is-better-than-telegram.md:36]: Slightly ambiguous phrasing where you say:

> Signal is not trying to be social media. I want my primary messaging app to be just that...

Sounds a bit like you're saying you want your primary messaging app to be social media, although this is obviously disambiguated by the second part of the sentence.

File: posts/signal-is-better-than-telegram.md (hunk 1 above)
Comment (ucg8j) [reply posts/signal-is-better-than-telegram.md]: https://github.com/ucg8j/blog/pull/2/commits/eb17bfd462501a8b87936b12c9727b08d3899879

I just went with capitalise all of the things!

File: posts/signal-is-better-than-telegram.md (hunk 2 above)
Comment (ucg8j) [reply posts/signal-is-better-than-telegram.md]: Ah yes, I wrote that whilst having a stroke.

https://github.com/ucg8j/blog/pull/2/commits/8c2aaf3f97fbe816f55a5edda13a75dc6d76dd4e

File: posts/signal-is-better-than-telegram.md (hunk 3 above)
Comment (ucg8j) [reply posts/signal-is-better-than-telegram.md]: This one took me a little while. But I finally got it, I think!
https://github.com/ucg8j/blog/pull/2/commits/0c63b6ce8053140021992300ec0f5a34e2e6ed17

**Pull request merged**